esrally/driver/driver.py

```diff
@@ -539,6 +542,16 @@ def num_cores(cfg):
     return int(cfg.opts("system", "available.cores", mandatory=False, default_value=multiprocessing.cpu_count()))
 
 
+ApiKey = collections.namedtuple("ApiKey", ["id", "secret"])
```

Comment:
Should we use a dataclass for this too?

Reply:
I thought about that, but since API keys are immutable, I opted to use a namedtuple instead. Do you think there's a significant advantage to using a dataclass instead?

Reply:
No significant advantage, I find named tuples a bit obscure and am not sure if [the performance issues were solved](https://lwn.net/Articles/731423/). Immutable is a nice advantage, though so let's keep things like this.